modules/microshift-ops-config-embed-ostree.adoc

```diff
@@ -0,0 +1,162 @@
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: cs-redhat-operator-index
+  namespace: openshift-marketplace <1>
+spec:
+  image: registry.example.com/redhat/redhat-operator-index:v4.15
+  sourceType: grpc
+  displayName:
+  publisher:
+  updateStrategy:
+    registryPoll:
+      interval: 60m
+----
+<1> The global namespace. Setting the `metadata.namespace` to `openshift-marketplace` enables the catalog to run in all namespaces. Subscriptions in any namespace can reference catalogs created in the `openshift-marketplace` namespace.
++
+[NOTE]
+====
+The default namespace security for `openshift-marketplace` is `baseline`, therefore a catalog source custom resource (CR) created in that namespace does not require a `spec.grpcPodConfig.securityContextConfig` value to be set. You can set a `legacy` or `restricted` value if required for the namespace and Operators you want to use.
+====
+
+. Get the SHA of the catalog index image by running the following command:
++
+[source,terminal]
+----
+$ skopeo inspect docker://registry.example.com/redhat/redhat-operator-index:v4.15 | jq `.Digest` <1>
+----
+<1> Pass the value from the `spec.image` field in the Catalog Source CR.
++
+.Example output
+[source,terminal]
+----
+"sha256:7a76c0880a839035eb6e896d54ebd63668bb37b82040692141ba39ab4c539bc6"
+----
+
+[IMPORTANT]
+====
+You must use the SHA instead of a tag in your catalog CR or the pod fails to start.
+====
+
+. Add the SHA of the catalog index commit to the Catalog Source (CR) by editing the YAML:
++
+.Example namespace `spec.image` configuration
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: cs-redhat-operator-index
+  namespace: openshift-marketplace
+spec:
+  image: registry.example.com/redhat/redhat-operator-index@sha256:7a76c0880a839035eb6e896d54ebd63668bb37b82040692141ba39ab4c539bc6 <1>
+  sourceType: grpc
+  displayName:
+  publisher:
+  updateStrategy:
+    registryPoll:
+      interval: 60m
+----
+<1> The SHA of the image commit.
+
+. Apply the YAML file from the results directory to the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ./oc-mirror-workspace/results-1708508014/catalogSource-cs-redhat-operator-index.yaml
+----
++
+.Example output
+[source,terminal]
+----
+catalogsource.operators.coreos.com/cs-redhat-operator-index created
+----
+
+.Verification
+
+. Verify that the `CatalogSource` resources were successfully installed by running the following command:
++
+[source,terminal]
+----
+$ oc get catalogsource --all-namespaces
+----
+//add example output
+
+. Verify that the catalog source is running by using the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-marketplace
+----
++
+.Example output
+[source,terminal]
+----
+NAME                             READY   STATUS    RESTARTS   AGE
+cs-redhat-operator-index-4227b   2/2     Running   0          2m5s
+----
```

Comment:
Using the verification label in the middle of procedure breaks the number. So, either remove the label and refactor the verification steps as sub-bullets or break this into two procedures.